src/printing.jl

```diff
@@ -5,7 +5,7 @@ const latex_subs = let
     repl_latex = Base.REPLCompletions.latex_symbols
     unicode = Dict([(v, k) for (k, v) in repl_latex])
     tex_name = Dict([(strip(k, '\\'), k) for (k, v) in repl_latex])
-    poppers = ["c", "k"]
+    poppers = [k for k in keys(tex_name) if length(k)==1]
```

Comment:
possibly, but I'm happily going to ignore that one for now because it sounds like it's kind of second order, isn't it ? Also I like the idea of having a list of poppers.